mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -154,6 +154,8 @@ func TestAccContainerCluster_misc(t *testing.T) {
 }
 
 func TestAccContainerCluster_withAddons(t *testing.T) {
+	// Multiple fine-grained resources cause VCR to fail
```

Comment:
This test and _withConfidentialNodes are both failing on the main branch already, even in non-VCR runs.

Unless these tests are known to be specifically impacted by this work, it would be great to revert these changes and fix the tests more holistically in a separate PR.

If these tests _are_ known to be impacted by this work, they will need to be fixed prior to merge.

Reply:
@melinath `_withAddons` and `_withConfidentialNodes` failures don't impact the changes in this PR, as this PR concentrates only on `gkehub membership config sync` and is independent of the features tested in the above 2 test cases.

Reply:
great, then all that needs to change is reverting the changes to those two test cases.